src/pages/store/checkout/contact-information.page.js

```diff
@@ -0,0 +1,479 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import CountrySelector from "../../../components/theme/checkout/country-selector.component";
+import StateSelector from "../../../components/theme/checkout/state-selector.component";
+import "./styles.css";
+import ck from "creditkey-js";
+import Checkout from "../../../components/checkout.component";
+
+function CheckoutPage() {
```

Comment:
the state being set here should use the same dynamic values that are used in what is now the dev component.